source/Government.cpp

```diff
@@ -216,6 +218,25 @@ void Government::Load(const DataNode &node)
 			enforcementZones.emplace_back(child);
 		else if(key == "provoked on scan")
 			provokedOnScan = true;
+		else if((child.Token(0) == "interdiction" || child.Token(0) == "interdiction bribe")
+				&& child.Size() >= 2)
+		{
+			const bool isInterdiction = child.Token(0) == "interdiction";
+			string &text = isInterdiction ? interdiction : interdictionBribe;
+			bool &seen = isInterdiction ? hasInterdiction : hasInterdictionBribe;
+
+			if(!seen)
+			{
+				text.clear();
+				seen = true;
+			}
+
+			const auto &value = child.Token(1);
```

Comment:
```suggestion
			const string &value = child.Token(1);
```
Usually we just get the string dont we?